core/src/test/kotlin/de/fraunhofer/iem/spha/core/strategy/OrKPICalculationStrategyTest.kt

```diff
@@ -0,0 +1,134 @@
+package de.fraunhofer.iem.spha.core.strategy
+
+import de.fraunhofer.iem.spha.core.hierarchy.KpiHierarchyNode
+import de.fraunhofer.iem.spha.model.kpi.KpiId
+import de.fraunhofer.iem.spha.model.kpi.KpiStrategyId
+import de.fraunhofer.iem.spha.model.kpi.RawValueKpi
+import de.fraunhofer.iem.spha.model.kpi.hierarchy.KpiCalculationResult
+import de.fraunhofer.iem.spha.model.kpi.hierarchy.KpiEdge
+import de.fraunhofer.iem.spha.model.kpi.hierarchy.KpiNode
+import kotlin.test.Test
+import org.junit.jupiter.api.Assertions.*
+
+class OrKPICalculationStrategyTest {
+
+    @Test
+    fun emptyEdges() {
+        assertEquals(
+            KpiCalculationResult.Empty(),
+            OrKPICalculationStrategy.calculateKpi(listOf(), strict = true),
+        )
+        assertEquals(
+            KpiCalculationResult.Empty(),
+            OrKPICalculationStrategy.calculateKpi(listOf(), strict = false),
+        )
+    }
+
+    @Test
+    fun calculateCorrectFalse() {
```

Comment:
I'm a bit confused with the test name. What is it trying to test?
I thought, based on the AND test, it is testing edges with correct = 100 and false != 100, but that's not the case with OR and XOR.

Reply:
oh well me too - I'll come up with a name that makes sense and commit a change later today